packages/trader/src/AppV2/Components/TradeParameters/TakeProfit/take-profit-input.tsx

```diff
@@ -44,13 +44,17 @@ const TakeProfitInput = React.forwardRef(
                         allowDecimals
                         disabled={!is_enabled}
                         decimals={decimals}
+                        data-testid='dt_input_with_steppers'
                         message={message}
+                        minusDisabled={Number(take_profit_value) - 1 <= 0}
```

Comment:
I request a couple of changes from Quill team for this component in order to expand it functionality in terms of validation and blocking some buttons. Hence, it's done in the latest version of Quill, I refactored Take Profit

Reply:
Oh boy what have they done to my component 😥